### PR TITLE
Update New hire checklist

### PR DESCRIPTION
**Pull request by TheAdamBorek**
Updated the new hire checklist after I've setup my new machine.
I've included Oliver's notes from the last PR https://github.com/Babylonpartners/ios-playbook/pull/73

### CODE REVIEW

File: TechnicalDocuments/NewHiresCheckList.md [hunk 1]
```diff
@@ -41,8 +41,6 @@ Here's how to get the iOS project up and running.
 1. Globally configure Git to use SSH instead of HTTPS:
      ```
      git config --global url."git@github.com:".insteadOf "https://github.com/"
-     git config --global url."git@github.com:".pushinsteadof "git://github.com/"
-     git config --global url."git@github.com:".pushinsteadof "https://github.com/"
```
Comment (TheAdamBorek) [inline TechnicalDocuments/NewHiresCheckList.md]: Those 2 are not needed. I only had to setup the first line and everything works just fine.

**Comment by AliSoftware** [comment]
Also I just saw that there seems to be an extra `\` in the beginning of the code-block in step 6 ("Install RVM and Ruby version 2.4"), if you could take the occasion of this PR to remove it that would be great

**Comment by TheAdamBorek** [comment]
@AliSoftware done

File: TechnicalDocuments/NewHiresCheckList.md [hunk 2]
```diff
@@ -54,15 +52,17 @@ Here's how to get the iOS project up and running.
...
+1. Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios  
+*Make sure you've already done the `Setup Git LFS` step before cloning the project.*
      ```
      git clone git@github.com:Babylonpartners/babylon-ios.git
      ```
 
 1. Install RVM and Ruby version 2.4
      ```
-     \curl -sSL https://get.rvm.io | bash -s stable
+     curl -sSL https://get.rvm.io | bash -s stable
```
Comment (mluisbrown) [inline TechnicalDocuments/NewHiresCheckList.md]: Why did you remove the leading `\` ? This is means it will really run `curl` and ignore any alias you might have.

Comment (TheAdamBorek) [reply TechnicalDocuments/NewHiresCheckList.md]: I didn't know that as I thought it's there by mistake. I believe @AliSoftware also thought so :D 
bringing it back

Comment (AliSoftware) [reply TechnicalDocuments/NewHiresCheckList.md]: Oh, TIL indeed.

Comment (mluisbrown) [reply TechnicalDocuments/NewHiresCheckList.md]: It was also TIL for me when I first saw this on the RVM site 😄

Comment (AliSoftware) [reply TechnicalDocuments/NewHiresCheckList.md]: @TheAdamBorek If we put it back, I thonk it would then be worth mentioning the trick using a `# The backslash ensures this command will run the real curl and not an alias` or something? as many would otherwise assume it is a typo
